packages/xo-server-sdn-controller/src/index.js

```diff
@@ -622,8 +641,25 @@ class SDNController extends EventEmitter {
     vni,
     encrypted,
     mtu,
+    preferredCenterId,
   }) {
-    const privateNetwork = new PrivateNetwork(this, uuidv4())
+    const preferredCenter =
+      preferredCenterId !== undefined
+        ? this._xo.getXapiObject(this._xo.getObject(preferredCenterId, 'host'))
+        : undefined
+
+    if (preferredCenter !== undefined) {
+      // Put pool of preferred center first
+      const orderedPoolIds = [preferredCenter.$pool.$id]
```

Comment:
How are they ordered?

Reply:
preferredCenter's pool's UUID first to be sure to add it first in the private networks.